sqlmesh/core/plan/definition.py

```diff
@@ -284,6 +284,10 @@ def is_selected_for_backfill(self, model_fqn: str) -> bool:
     def plan_id(self) -> str:
         return self.environment.plan_id
 
+    @property
+    def is_prod(self) -> bool:
```

Comment:
nit: seems unecessary

Reply:
The double negative of `not self._is_dev` all over the place to indicate `prod` was bothering me. But I can revert this if you prefer

Reply:
meh up to you